lib/galaxy/jobs/runners/cli.py

```diff
@@ -168,12 +168,13 @@ def check_watched_items(self):
                 state = job_interface.parse_single_status(cmd_out.stdout, external_job_id)
                 if not state == model.Job.states.OK:
                     log.warning(f'({id_tag}/{external_job_id}) job not found in batch state check, but found in individual state check')
+            job_state = ajs.job_wrapper.get_state()
```

Comment:
You can move this to line 160 and re-use it on line 161.

Reply:
This was actually intentional as the single status check in the intervening lines could take a while, so I was worried the status might be stale by here if it was fetched earlier.